### PR TITLE
Arthig/main whisper working

### PR DESCRIPTION
**Pull request by agokarn**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
+	TARGET_PATH ?= '/v1/audio/transcriptions'
+	SCORING_ENDPOINT ?= 'http://localhost:9443/score'
+else ifeq ($(MODEL), whisper_aoai_local)
+	TARGET ?= http://127.0.0.1:5001
+	TARGET_PATH ?= '/v1/engines/whisper/audio/transcriptions'
+	SCORING_ENDPOINT ?= 'http://localhost:9443/score'
+else ifeq ($(MODEL), whisper_aoai)
+	TARGET ?= http://127.0.0.1:5002
+	TARGET_PATH ?= '/v1/engines/whisper/audio/transcriptions'
+	DEPLOYMENT ?= 'arthig-deploy16'
```
Comment (kapilvgit) [inline Makefile:17]: We may want to avoid checking in such constants?

File: Makefile [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+	SCORING_ENDPOINT ?= 'https://arthig-ep.eastus2.inference.ml.azure.com/score'
```
Comment (kapilvgit) [inline Makefile:18]: Again, can we avoid committing there URLs?

File: Makefile (hunk 1 above)
Comment (agokarn) [reply Makefile:17]: We already have these and are useful for testing. We dont need to publish this for the customers

File: Makefile (hunk 2 above)
Comment (agokarn) [reply Makefile:18]: I feel they are very similar to KMS ?= https://acceu-aml-504.confidential-ledger.azure.com
MAA ?= https://maanosecureboottestyfu.eus.attest.azure.net

**Review by kapilvgit — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (agokarn) [reply Makefile:18]: Ahh now I see https://github.com/kapilvgit/ohttp/pull/14/files#diff-fce48f84d10d3f19153a9693851a1f3074102b37eda9ed3cf702d6019211bab1. How about either of us goes first and the other folds the relevant changes

**Pull request merged**